### ISSUE TITLE
$PATH on OS X El Capitan

### ISSUE THREAD

**Issue by ventsislaf** [thoughtbot/dotfiles#420]
It looks like `path_helper` on OS X El Capitan is moved from `/etc/zshenv` to `/etc/zprofile` which is overriding the `$PATH` after the user's customizations.
More info: http://www.zsh.org/mla/users/2015/msg00727.html

I'm not sure if this has to be addressed in [thoughtbot/dotfiles](https://github.com/thoughtbot/dotfiles) or [thoughtbot/laptop](https://github.com/thoughtbot/laptop).

**Comment by ventsislaf** [comment]
As a fast workaround I moved this code

```sh
# ensure dotfiles bin directory is loaded first
export PATH="$HOME/.bin:/usr/local/sbin:$PATH"

# load rbenv if available
if which rbenv &>/dev/null ; then
  eval "$(rbenv init - --no-rehash)"
fi

# mkdir .git/safe in the root of repositories you trust
export PATH=".git/safe/../../bin:$PATH"
```

from `~/.zshenv` to `~/.zprofile`.

**Comment by geoffharcourt** [comment]
@ventsislaf, I'm seeing the same thing. Looking at a solution that will work across Yosemite and El Capitan.

**Comment by geoffharcourt** [comment]
Fixed in 15de0bcfee0d7a6472da048029408c5faedbae4e through documentation. Thank you for bringing this to our attention.
